test/test_distributions.py

```diff
@@ -338,20 +370,32 @@ def test_unit(batch_shape):
 
 @pytest.mark.parametrize('jax_dist, sp_dist, params', CONTINUOUS)
 def test_sample_gradient(jax_dist, sp_dist, params):
-    if not jax_dist.reparametrized_params:
+    # we have pathwise gradient for gamma sampler
+    gamma_derived_params = {
+        "Gamma": ["concentration"],
+        "Beta": ["concentration1", "concentration0"],
+        "Chi2": ["df"],
+        "InverseGamma": ["concentration"],
+        "LKJ": ["concentration"],
+        "LKJCholesky": ["concentration"],
+        "StudentT": ["df"]
+    }.get(jax_dist.__name__, [])
+    reparameterized_params = [p for p in jax_dist.reparametrized_params
```

Comment:
Is there a reason to not test these?

Reply:
I think this test is not designed for pathwise gradient:
```python
import torch
import pyro
import pyro.distributions as dist

def f(x):
    pyro.set_rng_seed(0)
    return dist.Gamma(x, 1).rsample()

x = torch.tensor(1., requires_grad=True)
f(x).backward()
eps = 1e-3
numerical = (f(x + eps) - f(x - eps)) / (2 * eps)
print(x.grad, numerical.detach())
```
print `tensor(1.9066) tensor(1422.4575)`.

Reply:
We have a separate test `test_pathwise_gradient` below, but it only works for `gamma` and `chi2`. Other distributions have high variance.

Reply:
Thanks for explaining, @fehiepsi! LGTM.